web/src/input_pill.ts

```diff
@@ -341,10 +341,7 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
 
             // if the user backspaces and there is input, just do normal char
             // deletion, otherwise delete the last pill in the sequence.
-            if (
-                e.key === "Backspace" &&
-                (funcs.value(e.target).length === 0 || window.getSelection()?.anchorOffset === 0)
-            ) {
+            if (e.key === "Backspace" && funcs.value(e.target).length === 0) {
```

Comment:
So I used `git log -SanchorOffset` and found b57ebe717effc920637d3e5853205729cdb4288d to figure out why that code was present. And I'm pretty sure this just re-introduces the bug fixed there.